internal/log_analysis/log_processor/registry/registry.go

```diff
@@ -77,6 +78,20 @@ var (
 			&zeeklogs.ZeekDNS{}, zeeklogs.ZeekDNSDesc),
 		(&suricatalogs.AnomalyParser{}).LogType(): DefaultLogParser(&suricatalogs.AnomalyParser{},
 			&suricatalogs.Anomaly{}, suricatalogs.AnomalyDesc),
+		(&gitlablogs.APIParser{}).LogType(): DefaultLogParser(&gitlablogs.APIParser{},
+			&gitlablogs.API{}, gitlablogs.APIDesc),
+		(&gitlablogs.RailsParser{}).LogType(): DefaultLogParser(&gitlablogs.RailsParser{},
+			&gitlablogs.Rails{}, gitlablogs.RailsDesc),
+		(&gitlablogs.IntegrationsParser{}).LogType(): DefaultLogParser(&gitlablogs.IntegrationsParser{},
+			&gitlablogs.Integrations{}, gitlablogs.IntegrationsDesc),
+		(&gitlablogs.GitParser{}).LogType(): DefaultLogParser(&gitlablogs.GitParser{},
+			&gitlablogs.Git{}, gitlablogs.GitDesc),
+		(&gitlablogs.AuditParser{}).LogType(): DefaultLogParser(&gitlablogs.AuditParser{},
+			&gitlablogs.Audit{}, gitlablogs.AuditDesc),
+		(&gitlablogs.ExceptionsParser{}).LogType(): DefaultLogParser(&gitlablogs.ExceptionsParser{},
+			&gitlablogs.Exceptions{}, gitlablogs.ExceptionsDesc),
+		(&awslogs.CloudTrailInsightParser{}).LogType(): DefaultLogParser(&awslogs.CloudTrailInsightParser{},
+			&awslogs.CloudTrailInsightParser{}, awslogs.CloudTrailInsightDesc),
```

Comment:
Should be: 
```suggestion
		(&awslogs.CloudTrailInsightParser{}).LogType(): DefaultLogParser(&awslogs.CloudTrailInsightParser{},
			&awslogs.CloudTrailInsight{}, awslogs.CloudTrailInsightDesc),
```
That's the reason the generation of the table is failing

Reply:
Nice catch, fixing